tetrate/check-image-FIPS-compliant copy/check-image-FIPS-compliant.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: enforce-fips-compliant-images
+  annotations:
+    policies.kyverno.io/title: "Enforce FIPS Compliant Images"
+    policies.kyverno.io/category: "Security"
+    policies.kyverno.io/severity: "high"
+    policies.kyverno.io/subject: "Pod, Deployment, StatefulSet, DaemonSet"
+    policies.kyverno.io/description: "Ensures that container images are pulled from the FIPS-compliant registry fips-containers.istio.tetratelabs.com."
+spec:
+  validationFailureAction: Enforce
+  background: true
+  rules:
+    - name: check-fips-registry
+      match:
+        any:
+          - resources:
+              kinds:
+                - Pod
+                - Deployment
```

Comment:
We should probably write the check only for `Pod`. Other controllers will be handled by auto-gen rules.